src/main/java/eu/merloteducation/organisationsorchestrator/service/ParticipantService.java

```diff
@@ -336,6 +336,16 @@ public MerlotParticipantDto createParticipant(RegistrationFormContent registrati
             metaDataDto);
     }
 
+    /**
+     * Return the list of trusted dids. In the context of MERLOT the dids of the federators are trusted.
+     *
+     * @return list of trusted dids
+     */
+    public List<String> getTrustedDids() {
+        List<MerlotParticipantMetaDto> metadataList = organizationMetadataService.getParticipantsByMembershipClass(MembershipClass.FEDERATOR);
+        return metadataList.stream().map(MerlotParticipantMetaDto::getOrgaId).toList();
```

Comment:
Please fetch the ids on repo level such that not the whole metadata objects are returned by the database